gluten-flink/runtime/src/main/java/org/apache/flink/streaming/runtime/translators/SourceTransformationTranslator.java

```diff
@@ -81,12 +82,29 @@ private Collection<Integer> translateInternal(
     final ExecutionConfig executionConfig = streamGraph.getExecutionConfig();
 
     // --- Begin Gluten-specific code changes ---
-    if (transformation.getSource().getClass().getSimpleName().equals("NexmarkSource")) {
+    Class<?> sourceClazz = transformation.getSource().getClass();
+    if (sourceClazz.getSimpleName().equals("NexmarkSource")) {
```

Comment:
Just curious why reflection is needed here? Is it because the released velox4j jar doesn't include these source files?